### DISCUSSION TITLE
private by default

### DISCUSSION THREAD

**Discussion by jpw1991** [community/community#29076] [Other Feature Feedback, Questions, & Ideas]
I was just going through my repository list and saw that one of my repositories that should've been private, was public, and had been that way for months. Pretty annoying & not nice at all. I'm fortunate that I had externalized my OAuth keys to file because if these had been committed, anyone could have them.

For the sake of safety, why not make private the default option? Or at least, it'd be nice to have it the default option _for me_, if not for everyone.

**Comment by jfullstackdev** [comment]
what happened to the repo, from private to public ? never had that experience

**Reply by jpw1991** [reply]
I migrated a private repo from bitbucket to github and didn't realize it would become public.
